util/stellar_units.py

```diff
@@ -33,6 +33,7 @@ def units_to_stroops(amount, numeric_representation=False):
     amount = str(amount)
     if DECIMAL_POINT in amount:
         integer_part, fractional_part = amount.split(DECIMAL_POINT)
+        fractional_part = fractional_part[:DEC] if len(fractional_part) > DEC else fractional_part
```

Comment:
You do not need the if here. It's enough to say `fractional_part = fractional_part[:DEC]`. It's also faster :)